kbase-extension/static/kbase/js/common/jobCommChannel.js

```diff
@@ -300,14 +299,9 @@ define([
                     const jobIdList = msgData[BE_PARAMS.JOB_ID]
                         ? [msgData[BE_PARAMS.JOB_ID]]
                         : msgData[BE_PARAMS.JOB_ID_LIST];
-                    if (msgData.source === REQUESTS.LOGS) {
-                        msgTypeToSend = RESPONSES.LOGS;
-                    } else {
-                        msgTypeToSend = RESPONSES.ERROR;
-                    }
```

Comment:
if the logs cannot be found (e.g. when a job is in the queue), the BE now returns a RESPONSES.LOGS message instead of an error message ==> this workaround can be removed.